experimental/dds/tree2/src/test/snapshots/files/has-handle.json

```diff
@@ -27,7 +27,7 @@
                     "tree": {
                         "ForestTree": {
                             "type": 2,
-                            "content": "[{\"type\":\"Handle\",\"value\":{\"type\":\"__fluid_handle__\",\"url\":\"/test/test\"}}]"
+                            "content": "[[\"rootFieldKey\",[{\"type\":\"Handle\",\"value\":{\"type\":\"__fluid_handle__\",\"url\":\"/test/test\"}}]]]"
```

Comment:
now that empty root fields work some what differenty than non-empty ones, make sure we have test coverage for the empty case.